detection-rules/link_ludus_presentation_cred_theft.yml

```
@@ -0,0 +1,104 @@
+name: "Link: Ludus Presentation With Credential Phishing Indicators"
```

stay in sync for names of similar rules
```suggestion
name: "Link: Multistage Landing - Ludus Presentation"
```